api/tests/acceptance/application/campaign-controller_test.js

```diff
@@ -1278,4 +1278,70 @@ describe('Acceptance | API | Campaign Controller', function () {
       expect(response.statusCode).to.equal(400);
     });
   });
+
+  describe('PATCH /api/campaigns/{id}', function () {
+    it('should return 200 when user is admin or owner of the campaign', async function () {
+      // given
+      const organization = databaseBuilder.factory.buildOrganization();
+      const campaign = databaseBuilder.factory.buildAssessmentCampaign({ organizationId: organization.id });
```

Comment:
ce build crée un ownerId et  un creatorId, l'admin est pas du tout owner de la campagne, je suggère de renommer le test en en `should return 200 when user is admin but not owner of the campaign`